scripts/install.ps1

```diff
@@ -175,6 +175,15 @@ function InstallMinicondaPip ($python_home) {
 function main () {
     InstallPython $env:PYTHON_VERSION $env:PYTHON_ARCH $env:PYTHON
     InstallPip $env:PYTHON
+
+    (new-object net.webclient).DownloadFile("http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.3/PyQt4-4.11.4-gpl-Py3.4-Qt4.8.7-x32.exe", "C:\install-PyQt4-py34.exe")
+    Start-Process -FilePath C:\install-PyQt4-py34.exe -ArgumentList "/S" -Wait -Passthru
+			
+	(new-object net.webclient).DownloadFile("http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.4/PyQt4-4.11.4-gpl-Py2.7-Qt4.8.7-x32.exe", "C:\install-PyQt4-py27.exe")
+    Start-Process -FilePath C:\install-PyQt4-py27.exe -ArgumentList "/S" -Wait -Passthru
+    
+    (new-object net.webclient).DownloadFile("http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.4/PyQt4-4.11.4-gpl-Py3.4-Qt5.5.0-x32.exe", "C:\install-PyQt5-py34.exe")
```

Comment:
Late night coding. :weary: 

I think I will put this logic in `script/install-qt.py`, similar to what you did in `qutebrowser`. :wink: